docs/source/v1/WaterCycle/simulation_data/index.rst

```diff
@@ -0,0 +1,62 @@
+***************
```

Comment:
This page can be seen in rendered form at https://portal.nersc.gov/cfs/e3sm/forsyth/data_docs_59/html/v1/WaterCycle/simulation_data/index.html.

It is essentially the [v2 page](https://portal.nersc.gov/cfs/e3sm/forsyth/data_docs_59/html/v2/WaterCycle/simulation_data/index.html) with a few edits.

Reply:
Looks fine, but I haven't tried the commands listed.